INN/umbrella-sfpublicpress
# Global nav and header search

## Changes

This pull request makes the following changes:

- Rearranges the global nav partial so that the search comes before the donate and social buttons
- Styles the search based on the compromise styles in https://github.com/INN/umbrella-sfpublicpress/issues/12
- Styles the buttons and other things based on https://github.com/INN/umbrella-sfpublicpress/issues/11

![Screen Shot 2020-01-09 at 18 14 35](https://user-images.githubusercontent.com/1754187/72112891-f28a5e80-330c-11ea-9c0d-bb6dc0e24777.png)
![Screen Shot 2020-01-09 at 18 18 36](https://user-images.githubusercontent.com/1754187/72112892-f28a5e80-330c-11ea-85df-1993e9a4cd90.png)
![Screen Shot 2020-01-09 at 18 19 05](https://user-images.githubusercontent.com/1754187/72112893-f28a5e80-330c-11ea-8cf5-157437f21e64.png)
![Screen Shot 2020-01-09 at 18 19 14](https://user-images.githubusercontent.com/1754187/72112894-f28a5e80-330c-11ea-9923-b40f2ccb67eb.png)
![Screen Shot 2020-01-09 at 18 19 24](https://user-images.githubusercontent.com/1754187/72112895-f322f500-330c-11ea-8578-1b4969226b6f.png)
![Screen Shot 2020-01-09 at 18 19 29](https://user-images.githubusercontent.com/1754187/72112898-f322f500-330c-11ea-8d08-0657248a9387.png)


## Why



Resolves https://github.com/INN/umbrella-sfpublicpress/issues/11
Resolves https://github.com/INN/umbrella-sfpublicpress/issues/12

## Testing/Questions

Features that this PR affects:

- global nav on not-mobile on all pages


Questions that need to be answered before merging:

- [x] Do the search styles look okay? Should we remove the white border?
- [x] This does not implement the "Hot topics:" label on the left. If we want to do that, it's going to be a mess to implement, especially if it's supposed to be user-editable. https://github.com/INN/umbrella-sfpublicpress/issues/11#issuecomment-571814972 (well, we could make the first item in that menu italic, but then the first item there would also show up in the mobile nav. It's a bad hack.)

Steps to test this PR:

1. check out the branch
2. Put some menu items in the Global Nav menu
3. In Theme Options > Basic Settings:
    - Show the donation button, give it a link
    - Set the social media URLs
    - Set the social media buttons to appear in the header
